flybywiresim/a32nx
# fix: Power On Self Test fixes for PFD/ND/MFD

Fixes #945

## Summary of Changes

- [x] When powered up and ND brightness knob is turned off,  PFD now waits for 10 seconds before Power On Self-Test is restarted
- [x] When powered up and ND brightness knob is turned off,  MFD now waits for 10 seconds before Power On Self-Test is restarted
- [x] When  powered up and Lower ECAM brightness knob is turned off, Lower ECAM now waits for 10 seconds before Power On Self-Test is restarted
- [x] When powered up and Upper ECAM brightness knob is turned off, Lower ECAM now waits for 10 seconds before Power On Self-Test is restarted

## References
https://discordapp.com/channels/738864299392630914/754851312306487326/758257387432968222
See #945 




## Additional context



Discord username (if different from GitHub):

## Testing instructions


Power on test for each display (PFD/MFD/Lower ECAM/Upper ECAM):

- Power-up Aircraft
  - Observe and wait for Power On Self-Test (POST) for Displays to Complete
  - Turn Brightness knob off for less than 10 seconds, then turn display back on
    - Display should continue to display relevant information.
  - Turn Brightness knob off for at least 10 seconds, then turn display back on.
    - Display should now display Self-Test
  
For each display, testing by switching each display in turn to various on/off states:
 - Power-off Aircraft
   - Displays should stop displaying information and go blank
 - Power-on aircraft again
   - Display should begin self-test as expected

Displays should work normally as per normal flight operations in every case in flight.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
